### DISCUSSION TITLE
TF activators

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1479] [Questions / Documentation]
Hi @rcorces,

I was wondering if you think it's possible to know whether a TF is an activator or a suppressor from ATAC-seq data, since TF functionality is co-factor dependent and highly context-specific. 

If it's somewhat possible (albeit not 100%), how would you go about determining it in ArchR?

Thanks a lot for your opinion!

**Comment by rcorces** [comment]
I'm not aware of any methods to *know* if its an activator or suppressor from ATAC data alone. You can, of course, look at co-accessibility and look for a negative correlation but this is far from certainty.
